roles/oneview_id_pools_ipv4_subnet/defaults/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+# defaults file for oneview_id_pools_ipv4_subnets
+config: "~/.ansible/collections/ansible_collections/hpe/oneview/roles/oneview_id_pools_ipv4_subnets/files/oneview_config.json"
```

Comment:
check the path properly. You have mentioned the role name as ```oneview_id_pools_ipv4_subnet``` singular but the path to config file is showing ```oneview_id_pools_ipv4_subnets``` plural